example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -185,6 +184,7 @@
 				TargetAttributes = {
 					97C146ED1CF9000F007C117D = {
 						CreatedOnToolsVersion = 7.3.1;
+						DevelopmentTeam = S38X8U35U5;
```

Comment:
Who this?

Reply:
That's my personal development team. Seems like this file must be committed.
Let me know if there is a way to ignore this.

Note: this is project file in example app

Reply:
@QuintinWillison do we need to commit these XCode files?  I assume we do given this is an example app and we want it to just work, but ideally we'd not commit this stuff.

Reply:
Yes, you do need to commit them. They are required for the Xcode project to build and they cannot be automatically built unless you start employing other tech like make which is not idiomatic for iOS devs. Xcode project files are standard currency in Apple development.

Reply:
@tiholic my preference would be to remove these lines from this file and then just be mindful whenever you commit changes to this file to make sure you omit chunks that are personalised like this. We can discuss offline how to handle this when doing iOS development (short answer: nice tooling, e.g. [SourceTree](https://www.sourcetreeapp.com/)).